Update reflections-of-light

jswan5 opened the pull request:
Simplify notebook by removing interactivity for multiple choice questions. Modify code in helper file to support interactive features.

Removing the interactivity for the MC questions hides the answers, but a table at the end contains them. Hopefully that's still sufficient?

misterhay commented:
Looks good, the interactives are all working.

The multiple choice questions can be removed entirely, both from the markdown cells and from the `helper.py` file.

There are also some unused imports in `helper.py` that can be cleaned up.

I wonder if the `matplotlib` functions are simple enough that we can move them into code cells and delete `helper.py`, so that students can more easily see how they are being generated.

jswan5 commented:
@misterhay, I agree with all your changes, and I removed all the MC question cells and moved the code from the helper.py file to inline. Functions work well and now the code is visible. 

Do you want to check out the wording and let me know if you think it needs further changes?

misterhay reviewed (approved):
I think this is ready for merge
